File: planner/app/src/main/java/com/sunilson/firenote/data/models/Note.java
package com.sunilson.firenote.data.models;

/**
 * @author Linus Weiss
 */

public class Note extends Element{

    private String text;

    public Note(String noteType, String title) {
        super(noteType, title);
        this.text = "";
    }

    public Note() {
        super();
    }

    public void setText(String text) {
        this.text = text;
    }

    public String getText() {
        return text;
    }
}

File: planner/app/src/main/java/com/sunilson/firenote/data/models/Note.kt
package com.sunilson.firenote.data.models

import java.util.*

class Note( var text: String,
            elementID: String,
            category: Category,
            noteType: String,
            color: Int,
            locked: Boolean,
            timestamp: Long = Date().time,
            title: String = "New Element") : Element(elementID, category, noteType, color, locked, timestamp, title) {

}